nixos/tests/common/ec2.nix

```diff
@@ -36,6 +37,8 @@ with pkgs.lib;
                 "create",
                 "-f",
                 "qcow2",
+                "-F",
+                "qcow2",
```

Comment:
Modification of this file should go in a separate commit to keep clean history

Reply:
I moved these changes to another commit in the same PR. Should I move them to another PR?

Reply:
No, but the commit should be formatted using a path name: `nixos/tests/ec2` is better

Reply:
Okay, thanks, I wasn't sure quite how to format it.